tuf/scripts/repo.py

```diff
@@ -514,19 +526,11 @@ def add_verification_key(parsed_arguments):
 
     if parsed_arguments.role not in ('root', 'targets', 'snapshot', 'timestamp'):
       raise exceptions.Error('The given --role is not a top-level role.')
-
-    elif parsed_arguments.role == 'root':
-      repository.root.add_verification_key(imported_pubkey)
-
-    elif parsed_arguments.role == 'targets':
-      repository.targets.add_verification_key(imported_pubkey)
-
-    elif parsed_arguments.role == 'snapshot':
-      repository.snapshot.add_verification_key(imported_pubkey)
-
-    # The timestamp role..
     else:
-      repository.timestamp.add_verification_key(imported_pubkey)
+      role = getattr(repository, parsed_arguments.role)
+      role.add_verification_key(imported_pubkey)
+      if role.threshold:
```

Comment:
Why the if? This is not obvious at all.

Reply:
[Explanation](https://github.com/theupdateframework/tuf/pull/1473#discussion_r672829998)